cnd/src/http_api/routes/index.rs

```diff
@@ -182,32 +190,66 @@ impl From<Body<HanEthereumEther, HalightLightningBitcoin>>
             ethereum_amount: body.alpha.amount,
             lightning_identity: body.beta.identity,
             lightning_cltv_expiry: body.beta.cltv_expiry.into(),
-            lightning_amount: body.beta.amount.0,
+            bitcoin_amount: body.beta.amount.0,
         }
     }
 }
 
 #[derive(serde::Deserialize, Clone, Debug)]
-pub struct HanEthereumEther {
+struct Hneth {
     pub amount: asset::Ether,
     pub identity: identity::Ethereum,
     pub chain_id: u32,
     pub absolute_expiry: u32,
 }
 
+impl From<Hneth> for hneth::CreatedSwap {
+    fn from(p: Hneth) -> Self {
+        hneth::CreatedSwap {
+            amount: p.amount,
+            identity: p.identity,
+            chain_id: p.chain_id,
+            absolute_expiry: p.absolute_expiry,
+        }
+    }
+}
+
 #[derive(serde::Deserialize, Clone, Debug)]
-pub struct HalightLightningBitcoin {
-    pub amount: Http<asset::Lightning>,
+struct Hlnbtc {
```

Comment:
```suggestion
struct HlnBtc {
```

Reply:
In accordance with @D4nte's change above I would actually vote for `HalightBtc`.